openwebwork/webwork2
# New User Calendar sidebar

This introduces a new sidebar that allows one to select users and then changes dates for a userSet via the calendar. 

To test
1: go to the calendar, select the "Users' Calendar" sidebar.
2. select one of more users.  Common sets will now be shown on the calendar.
3. Change dates and they will changes for all selected users.  You will see changes in the Problem Sets Details view, overrides tab.  
4. try filtering users via the textbox. 

Question:  Right now, there is no indication that the sets on the calendar are global or users other than the selected users in the sidebar.  If anyone has an idea about showing this, let me know. 

Notes:  
1. Most of the changes are due to the ww3-feature/back-button merge.  Please merge that pull request first. 
2. There is still a jquery-ui error that occurs when dragging/dropping.  It appears that this might be a bug and is fixed in jqueryui-1.11.  I will test this shortly.
